### ISSUE TITLE
Missing Function.JSON File?

### ISSUE THREAD

**Issue by tsull360** [dsanchezcr/ContactFormWithAzure#1]
Hi,
There doesn't seem to be a function.json file such that the solution can be deployed with vs code. Is that expected?

**Comment by ghost** [comment]
Not sure, did you successfully make this function work in your environment?

**Comment by tsull360** [comment]
No, not specifically this project. I did find another one that met my needs and was able to get it deployed and working.

**Comment by ghost** [comment]
I just figured out how to make the API work, you don't need a function.json, and you can put the environmental variables in the  local.settings.json file to make it work. I am working on a blazor webassembly project tonight to make the project fully functional.

**Comment by ghost** [comment]
Can you share the working repo with me just for reference?

**Comment by tsull360** [comment]
This is what I got going: https://github.com/mikepfeiffer/azure-serverless-contact

**Comment by ghost** [comment]
Thank you.

**Comment by ghost** [comment]
My version of the github repo is working properly and here it is for your reference:
https://github.com/Robertblazor/Contact-us-exp

Hope it can help you in the future.
